decidim-core/app/controllers/decidim/newsletters_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Decidim
+  # The controller to handle the user's notifications dashboard.
+  class NewslettersController < Decidim::ApplicationController
+    skip_authorization_check
+
+    layout "decidim/mailer"
+    helper Decidim::SanitizeHelper
+    include Decidim::NewslettersHelper
+
+    def show
+      @organization = current_organization
+      @newsletter = collection.find(params[:id])
```

Comment:
Move this to a helper method? If you have a typo on an instance variable, Ruby doesn't complain, but if you have a typo on a method call then it does. That's why I prefer using helper methods to hide those instance variables, so that we avoid possible future bugs due to typos 😄